webpack.config.dev.js

```diff
@@ -22,8 +22,24 @@ module.exports = {
   module: {
     loaders: [{
       test: /\.js$/,
-      loaders: ['babel'],
-      include: path.join(__dirname, 'source')
+      include: path.join(__dirname, 'source'),
+      loader: 'babel',
+      query: {
```

Comment:
I meant not to use `query` for babel loader at all. Combining loader `query` and `.babelrc` is definitely not something I would recommend.